packages/xo-web/src/common/intl/messages.js

```diff
@@ -756,9 +756,13 @@ const messages = {
   addSrLabel: 'Add SR',
   addVmLabel: 'Add VM',
   addHostLabel: 'Add Host',
-  hostNeedsPatchUpdate:
-    'This host needs to install {patches, number} patch{patches, plural, one {} other {es}} before it can be added to the pool. This operation may be long.',
-  hostNeedsPatchUpdateNoInstall:
+  missingPatchesWarining:
+    'The missing patches need to be installed. This operation may be long.',
+  missingPatchesPool:
+    'The pool needs {nMissingPatches, number} patch{nMissingPatches, plural, one {} other {es}}.',
+  missingPatchesHost:
+    'This host needs {nMissingPatches, number} patch{nMissingPatches, plural, one {} other {es}}.',
+  patchUpdateNoInstall:
```

Comment:
By changing the key, you're invalidating all the translations.

Reply:
I even changed the content of the message